### PR TITLE
Add slave offset metrics

### PR DESCRIPTION
**Pull request by S3RK**
Expose offset of connected slaves, useful to monitor replication lags

### CODE REVIEW

**Comment by paunin** [comment]
+

File: exporter/redis.go [hunk 1]
```diff
@@ -334,6 +340,36 @@ func parseDBKeyspaceString(db string, stats string) (keysTotal float64, keysExpi
 	return
 }
 
+/*
+	slave0:ip=10.254.11.1,port=6379,state=online,offset=1751844676,lag=0
+	slave1:ip=10.254.11.2,port=6379,state=online,offset=1751844222,lag=0
+*/
+func parseConnectedSlaveString(slaveName string, slaveInfo string) (offset float64, ip string, state string, ok bool) {
+	ok = false
+	if matched, _ := regexp.MatchString(`^slave\d+`, slaveName); !matched {
```
Comment (oliver006) [inline exporter/redis.go]: Use `strings.HasPrefix()` instead?

File: exporter/redis_test.go [hunk 2]
```diff
@@ -552,7 +584,7 @@ func TestHTTPEndpoint(t *testing.T) {
 		`test_instance_info`,
 
 		// labels and label values
-		`addr="redis://localhost:6379"`,
+		`addr="redis://` + *redisAddr,
```
Comment (oliver006) [inline exporter/redis_test.go]: 👍

File: exporter/redis.go (hunk 1 above)
Comment (S3RK) [reply exporter/redis.go]: That was my first approach, but unfortunately, there are other parameters with this prefix.
For example
```
root@test-redis-node2-0:/data# redis-cli info | grep "^slave"
slave_repl_offset:79766288846
slave_priority:100
slave_read_only:1
```

Comment (oliver006) [reply exporter/redis.go]: ahh, ok, thanks, that makes sense.

**Pull request merged**